eslint-rules/tests/lib/rules/prop-value-shape-deprecation.js

```diff
@@ -0,0 +1,116 @@
+const RuleTester = require('eslint').RuleTester;
```

Comment:
Some general comments on the tests:
1. Add more, including more valid ones (I don't mind adding as well if you want me to).
2. It was not obvious what's the difference between the code and the output, consider moving `imageSource` and `source` to a couple of `const`s.

Reply:
1. I've added 2 more valid examples, feel free to suggest more :) 
2. Moved it to `const`s.